beacon_node/lighthouse_network/src/rpc/methods.rs

```diff
@@ -243,24 +244,24 @@ pub struct OldBlocksByRangeRequest {
 }
 
 /// Request a number of beacon block bodies from a peer.
-#[derive(Clone, Debug, PartialEq)]
+#[derive(Encode, Decode, Clone, Debug, PartialEq)]
 pub struct BlocksByRootRequest {
     /// The list of beacon block bodies being requested.
     pub block_roots: VariableList<Hash256, MaxRequestBlocks>,
 }
 
+/// Container of the data that identifies an individual blob. This is a P2P spec type.
+#[derive(Encode, Decode, Clone, Debug, PartialEq)]
```

Comment:
Should we move this to the consensus types crate? I know it's weird cause this is defined in the p2p part of the spec, but `BlobSidecar` and `SignedBlobSidecar` are as well. I think to work with blobs in consensus people with usually want this type too (for example to index cached blobs)

Reply:
oh I assumed this was only p2p because in other places the root and index are flattened, but I'm fine with each

Reply:
if it's necessary in other places, let's move it